packages/react-router-dom/package.json

```diff
@@ -46,7 +46,7 @@
   },
   "dependencies": {
     "@babel/runtime": "^7.3.1",
-    "@zumper/react-router": "^5.0.0-zumper.6",
+    "@zumper/react-router": "^5.0.1-zumper.7",
```

Comment:
This probably works but lerna kind of bumps the version number for you. When we go to publish we may prefer having an artificially deflated version. So, unpublished would be `5.0.1-zumper.6` and then we'd set the version to `5.0.1-zumper.7` when we run `lerna publish`